nibabel/streamlines/tests/test_tractogram.py

```diff
@@ -641,6 +687,11 @@ def test_lazy_tractogram_getitem(self):
         assert_raises(NotImplementedError,
                       DATA['lazy_tractogram'].__getitem__, 0)
 
+    def test_lazy_tractogram_extend(self):
+        t = DATA['lazy_tractogram'].copy()
+        new_t = DATA['lazy_tractogram'].copy()
+        assert_raises(NotImplementedError, new_t.__iadd__, t)
```

Comment:
Check `extend` method raises?   

Check add as well, using trick like the one above?

```
import operator
for op in (operator.add, operator.iadd, extender):
    assert_raises(NotImplementedError, op, new_t, t)
```